apps/dashboard/src/@/api/universal-bridge/links.ts

```diff
@@ -0,0 +1,43 @@
+import { DASHBOARD_THIRDWEB_SECRET_KEY } from "@/constants/server-envs";
```

Comment:
add a `import "server only";` to prevent this ever being used in a client component